### ISSUE TITLE
what about gdAlignRows, and other gd* attributes from flex-layout?

### ISSUE THREAD

**Issue by seyfer** [synopss/flex-layout-to-tailwind#9]
is it planned to support?
there is a hint: https://github.com/angular/flex-layout/issues/1426#issuecomment-1302184078

**Comment by synopss** [comment]
Yes, I didn't have much time to work on this lately as you can tell, but this CLI will support all attributes that are possible to migrate (because some aren't sadly).

**Comment by seyfer** [comment]
yep, this is a great starting tool, then what I do is
1. I keep the flex-layout package installed because some fx* attributes are either stay in the code or not converted properly
2. I go page by page and review the conversion, fixing and converting the remaining fx* attributes
3. also converting gd* manually
4. when all done - uninstall flex-layout

just having the gd* implemented would help a bit with this process. thanks!

**Comment by synopss** [comment]
> yep, this is a great starting tool, then what I do is
> 
>     1. I keep the flex-layout package installed because some fx* attributes are either stay in the code or not converted properly
> 
>     2. I go page by page and review the conversion, fixing and converting the remaining fx* attributes
> 
>     3. also converting gd* manually
> 
>     4. when all done - uninstall flex-layout
> 
> 
> just having the gd* implemented would help a bit with this process. thanks!

Can you write down there which fx attributes are not converted properly please ?

**Comment by seyfer** [comment]
@synopss I can give some examples, while I still migrate it at one of the projects. When I run CLI, these are pieces of code that I have left, and I go fix them manually.

```
<div
      fxLayoutAlign="start"
      fxLayoutAlign.md="center"
      fxLayoutAlign.sm="center"
      class="xs:flex xs:flex-col xs:justify-center xs:content-stretch xs:items-stretch xs:w-full flex flex-row"
    >
```

here probably only `fxLayoutAlign.xs="center"` was handled, but the rest left as it was before

```
<div
    mat-subheader
    gdAlignRows="stretch"
    class="job-header flex justify-between content-end items-end sm:flex sm:flex-col gap-[5px] h-full w-full min-h-full min-w-full"
    fxLayout="row"
    fxLayout.xs="column"
    fxLayoutAlign.xs="space-between"
    fxLayoutAlign.sm="space-between"
  >
```

```
<div
  gdAlignRows="stretch"
  fxLayout="row"
  fxLayout.xs="column"
  class="sm:flex sm:flex-col flex gap-2.5 h-full w-full min-h-full min-w-full"
>
```

hope this helps.

**Comment by synopss** [comment]
I see, so looks like it only handles the very first breakpoint they keep the rest.

**Comment by seyfer** [comment]
One more example. Converted but not removed

`<div fxShow.gt-md class="gt-md:basis-[35%] hidden gt-md:block">`
